test/e2e/fast-prototyping/README.md

```diff
@@ -0,0 +1,50 @@
+## Fast Prototyping
+
+This directory contains a set of scripts to run experiments for fast-prototyping cometbft reactors.
+Please note that (to date) these scripts are mostly oriented toward prototyping mempool reactors.
+
+For the impatient
+
+    cd path/to/cometbft/
+    E2E_DIR=$(pwd)/test/e2e
+    (cd ${E2E_DIR} && make node-fast generator runner docker-fast)
+    ${E2E_DIR}/fast-prototyping/experiments.sh ${E2E_DIR}/fast-prototyping/render/sample.csv # takes around two minutes
+    cp ${E2E_DIR}/fast-prototyping/render/results.csv.tmpl ${E2E_DIR}/fast-prototyping/render/results.csv
+    echo "sample;A sample result.;sample.csv" >>  ${E2E_DIR}/fast-prototyping/render/results.csv
+    ${E2E_DIR}/fast-prototyping/render.sh
+
```

Comment:
For the impatient, if you are running on Linux

```bash
cd path/to/cometbft/
E2E_DIR=$(pwd)/test/e2e
(cd ${E2E_DIR} && make node-fast generator runner docker-fast)
${E2E_DIR}/fast-prototyping/experiments.sh ${E2E_DIR}/fast-prototyping/render/sample.csv # takes around two minutes
cp ${E2E_DIR}/fast-prototyping/render/results.csv.tmpl ${E2E_DIR}/fast-prototyping/render/results.csv
echo "sample;A sample result.;sample.csv" >>  ${E2E_DIR}/fast-prototyping/render/results.csv
${E2E_DIR}/fast-prototyping/render.sh
```

If you are running on OSX, you might need to first install `coreutils`, e.g., by runnng `brew install coreutils`

Reply:
Thanks for finding this. I edited `README.md `with this information.